android/src/main/java/com/google/startupos/android/MainApplication.java

```diff
@@ -17,15 +17,29 @@
 package com.google.startupos.android;
 
 import android.app.Application;
+
+import java.io.IOException;
+
 import com.google.firebase.FirebaseApp;
+import android.util.Log;
+import org.json.JSONException;
 
-import com.google.bazel.example.android.FirestoreConfig;
+import com.google.startupos.android.FirestoreConfigLoader;
 
 public class MainApplication extends Application {
+
+  public static final String TAG = MainApplication.class.getSimpleName();
+
   @Override
   public void onCreate() {
     super.onCreate();
-    FirebaseApp.initializeApp(this, FirestoreConfig.getFirebaseConfig());
+    try {
```

Comment:
Why did we drop FirebaseApp.initializeApp()?

Reply:
@oferb we now delegate initialization to config loader

Reply:
If this also initializes and not only loads the config, let's call it FirestoreInitializer with init()